third-party/ios/ExecutorchLib/ExecutorchLib/Exported/ETModel.mm

```diff
@@ -34,38 +34,37 @@ - (NSArray *)forward:(NSArray *)input
            inputType:(NSNumber *)inputType {
   int inputTypeIntValue = [inputType intValue];
   std::vector<int> shapes = NSArrayToIntVector(shape);
-  ssize_t outputNumel = 0;
   @try {
     switch (inputTypeIntValue) {
     case 0: {
       // Int8Array
-      const int8_t *output =
-          runForwardFromNSArray<int8_t>(input, outputNumel, shapes, _model);
-      return arrayToNSArray<int8_t>(output, outputNumel);
+      std::vector<DataPtrWithNumel<int8_t>> output =
```

Comment:
I don't know how invasive this change would be, but changing `inputTypeIntValue` to enum type seems reasonable here

Reply:
This way, you would get rid of this magic numbers

Reply:
Good point, thanks for contributing:D